evadb/binder/statement_binder.py

```diff
@@ -194,12 +208,6 @@ def _bind_delete_statement(self, node: DeleteTableStatement):
 
     @bind.register(CreateTableStatement)
     def _bind_create_statement(self, node: CreateTableStatement):
-        # we don't allow certain keywords in the column_names
-        for col in node.column_list:
-            assert (
-                col.name.lower() not in RESTRICTED_COL_NAMES
-            ), f"EvaDB does not allow to create a table with column name {col.name}"
-
```

Comment:
I think we should not remove this. This causes testcase failures and we can not allow user to create columns with certain keywords.

Reply:
Thanks, updated it.